bicycle_steering_controller:
  wheelbase:
    {
      type: double,
      default_value: 0.0,
      description: "Wheelbase length.",
      read_only: false,
    }

  front_wheel_radius:
    {
      type: double,
      default_value: 0.0,
      description: "Front wheel radius.",
      read_only: false,
    }

  rear_wheel_radius:
    {
      type: double,
      default_value: 0.0,
      description: "Rear wheel radius.",
      read_only: false,
    }
